mbed_app.json

```diff
@@ -317,6 +317,34 @@
             "esp8266.cts"                               : "D12",
             "esp8266.rts"                               : "PB_1",
             "esp8266.rst"                               : "D8"
+        },
+        "WIO_3G": {
+            "target.features_add"                       : ["LWIP", "STORAGE", "BOOTLOADER"],
+            "target.macros_add"                         : ["MBEDTLS_USER_CONFIG_FILE=\"mbedTLSConfig_mbedOS.h\""],
+            "target.device_has_remove"                  : ["SERIAL_FC"],
+            "target.network-default-interface-type"     : "CELLULAR",
+            "target.components_add"                     : ["FLASHIAP"],
+            "target.bootloader_img"                     : "bootloader/mbed-bootloader-wio_3g-internal_flash-no_rot-v4.1.2.bin",
+            "target.header_offset"                      : "0x8000",
+            "target.app_offset"                         : "0x8400",
+            "nsapi.default-cellular-plmn"               : 0,
+            "cellular.use-apn-lookup"                   : false,
+            "cellular.debug-at"                         : false,
+            "lwip.ipv4-enabled"                         : true,
+            "lwip.ethernet-enabled"                     : false,
+            "lwip.ppp-enabled"                          : true,
+            "lwip.tcp-enabled"                          : true,
+            "lwip.pbuf-pool-size"                       : 16,
+            "lwip.mem-size"                             : 12500,
+            "update-client.bootloader-details"          : "0x08005ee8",
+            "update-client.application-details"         : "(MBED_ROM_START + MBED_BOOTLOADER_SIZE)",
+            "update-client.storage-address"             : "(MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_BASE_ADDRESS+MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_SIZE)",
+            "update-client.storage-size"                : "(1024*1024-MBED_CONF_STORAGE_TDB_INTERNAL_INTERNAL_SIZE)",
+            "update-client.storage-page"                : 1,
+            "mbed-cloud-client.update-storage"          : "ARM_UCP_FLASHIAP",
+            "storage_tdb_internal.internal_base_address": "(MBED_ROM_START+1024*1024)",
+            "storage_tdb_internal.internal_size"        : "(128*1024)",
```

Comment:
32 KiB would be enough.

Reply:
also makes nice symmetry with the bootloader-size.

Reply:
You are right.  `storage_tdb_internal.internal_size` should be 32kB (16*2) for the target.  I also updated the bootloader binary.